mos-platform/nes-mmc1/prg-rom-banked.ld

```diff
@@ -90,6 +150,8 @@ SECTIONS {
 
 PROVIDE(__prg_rom_size = 256);
 
+prg_rom_is_512 = __prg_rom_size >= 512 ? 1 : 0;
```

Comment:
Nit: this symbol needs to be prefixed `__` to meet the contract between a C hosted environment and a standards compliant C program. (Such symbol names are reserved by the C standard for use by the toolchain, i.e., us.)